src/Cms/HasFiles.php

```diff
@@ -72,8 +72,8 @@ public function file(
 		}
 
 		// find by global UUID
-		if (Uuid::is($filename, 'file') === true) {
-			return Uuid::for($filename, $this->$in())->model();
+		if ($uuid = Uuid::from($filename, 'file', $this->$in())) {
+			return $uuid->model();
```

Comment:
`Issue`: Same here, but not as critical. Here, the valid but non-existent UUID will make Kirby execute all of the other checks instead of exiting early (but before with an error like `Call to function model() on null`).

Reply:
That raises maybe a question: If something looks like a UUID but cannot be resolved to a model, would we rather have it throwing some form of error or just ignore it and try to resolve it with the other checks?

Reply:
I think we should throw an error. The UUID pattern is very unique (`file://something`) and it is quite unlikely that something else was meant. Especially not a filename as filenames cannot contain two slashes sequentially.

Reply:
Reverted it